contracts/scripts/test/smokeTest.sh

```diff
@@ -23,7 +23,7 @@
 # any child processes created by this process are killed once the main process is terminated
 trap "exit" INT TERM ERR
 trap "kill 0" EXIT
-nodeWaitTimeout=180
+nodeWaitTimeout=60
```

Comment:
Is this variable used?

Reply:
Line 53 in this file. Maybe github collapsed a few lines of this file when showing the diff

Reply:
Haha yea my bad, must be half asleep, thanks :)